app/views/agreements/index.html.erb

```diff
@@ -25,8 +24,10 @@
             <td><%= link_to agreement.title, agreement %></td>
             <td><%= truncate(agreement.content) %></td>
 
-            <td><%= link_to 'Edit', edit_agreement_path(agreement) %></td>
-            <td><%= link_to 'Destroy', agreement, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+            <td align="right">
+                <%= link_to 'Edit', edit_agreement_path(agreement), class: "btn btn-primary btn-sm" %> |
```

Comment:
I think you could use single quotes example:
 ```ruby
 <%= link_to 'Edit', edit_agreement_path(agreement), class: 'btn btn-primary btn-sm' %>
```
Once it hasn't strings interpolation.

Reply:
Yeah, sure

Reply:
Nice! Thanks!